server/singularity.yaml

```diff
@@ -0,0 +1,6 @@
+singularity:
+    command: singularity
+    init_command: \{command} .. (optional or default)
```

Comment:
Comment out

Reply:
init_command is commented out for singularity